src/menu.stories.ts

```diff
@@ -240,17 +240,17 @@ const meta: Meta = {
     context.canvasElement
       .querySelector('glide-core-menu')
       ?.addEventListener('click', (event: Event) => {
-        const isLink =
+        const link =
           event.target instanceof Element &&
           event.target.closest('glide-core-menu-link');
 
-        if (isLink && window.top) {
+        if (link && link.url === '/' && window.top) {
```

Comment:
If the URL is anything but `/`, then the Storybook user has changed the URL and wants to see Menu Link navigate to that page.